astroquery/mast/cutouts.py

```diff
@@ -104,10 +104,13 @@ def __init__(self):
         super().__init__()
 
         services = {"sector": {"path": "sector"},
-                    "astrocut": {"path": "astrocut"}}
+                    "astrocut": {"path": "astrocut"},
+                    "mt_sector": {"path": "moving_target/sector"},
+                    "mt_astrocut": {"path": "moving_target/astrocut"}
+                    }
         self._service_api_connection.set_service_params(services, "tesscut")
 
-    def get_sectors(self, coordinates=None, radius=0*u.deg, objectname=None):
+    def get_sectors(self, coordinates=None, radius=0*u.deg, objectname=None, moving_target=None, mt_type=None):
```

Comment:
Whatever gets included in `*` also needs to be added to `params = {"obj_id": moving_target}` in L157 before the service call right?

Reply:
`*` means that the rest of the parameters are keyword only, nothing additional is included in it.

Reply:
Oh I see - learned something new today. This might be something worth starting a new PR over, since its a syntax change, we'd want to add the `*` to all the Zcut methods as well, etc. What do you think?